client/actions/getMessages.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Action to get the messages belonging to a group
+ */
+
+import axios from 'axios';
+
+/**
+ * @return {promise} returns an array of messages
```

Comment:
Improper JSDoc format  it should contain function name and the return statement should be on the last line